cmake/dependencies.cmake

```diff
@@ -48,18 +48,8 @@ morpheus_utils_configure_cccl()
 # =================
 morpheus_utils_configure_rmm()
 
-# gflags
-# ======
-rapids_find_package(gflags REQUIRED
-  GLOBAL_TARGETS gflags
-  BUILD_EXPORT_SET ${PROJECT_NAME}-exports
-  INSTALL_EXPORT_SET ${PROJECT_NAME}-exports
-)
```

Comment:
Why are we removing gflags, aren't we using it?

Reply:
I moved the find to `morpheus_utils_configure_glog`.